samples/tfm_integration/tfm_secure_partition/sample.yaml

```diff
@@ -22,4 +22,4 @@ sample:
 
 tests:
   sample.tfm.secure_partition:
-    tags: tfm
+    tags: tfm trusted-firmware-m mcuboot
```

Comment:
convert to list please.

doesn't `tfm` stand for `trusted-firmware-m`?

Reply:
tfm does stand for trusted-firmware-m, but I don't know if people might have external tests or things that already use this tag. I need to add the trusted-firmware-m tag so that it matches the module name.